kubernetes/loculus/values.yaml

```diff
@@ -446,16 +446,19 @@ defaultOrganismConfig: &defaultOrganismConfig
         displayName: Isolate name
         header: Sample details
         ingest: ncbiIsolateName
+        enableSubstringSearch: true
       - name: authors
         displayName: Authors
         type: authors
         header: Authors
+        enableSubstringSearch: true
         truncateColumnDisplayTo: 15
         ingest: ncbiSubmitterNames
       - name: authorAffiliations
         displayName: Author affiliations
         generateIndex: true
```

Comment:
I think the `generateIndex` and `autocomplete` conflict with `enableSubstringSearch` - we need to disable them otherwise nothing changes afaict:
<img width="1011" alt="Brave Browser 2024-10-29 14 33 09" src="https://github.com/user-attachments/assets/8091da36-5088-4780-aeb9-6bb6319d2640">

Have you tested and it worked? Cause I can't see it working for author affiliations for example.

Reply:
Given the field types are currently exclusive/enum like, we could as well refactor them into a single field, something like: `searchType` with values like `substring | autocomplete | ...` (not sure what else we have, plain I guess, we can make the most common one the default) this would make our config a little bit more succinct and save us from misconfiguring.

Reply:
This doesn't need to be part of this PR I guess, just if you feel like it. We can also just move fast and iron out such things later.

But the values.yaml should of course work in practice :)

Reply:
I have fixed it! Have a look at the preview now.

I'd leave it at that for now.